Modules/Tokens/test/Tokens.API.Tests.Integration/Features/GetTokenById.feature

```diff
@@ -0,0 +1,54 @@
+﻿@Integration
+Feature: GetTokenById
+
+User requests a Token
+
+Scenario: Requesting an own Token as an authenticated user
+	Given the user is authenticated
+	And an own Token t
+	When a GET request is sent to the Tokens/{id} endpoint with t.Id
+	Then the response status code is 200 (OK)
+	And the response contains a Token
+
+Scenario: Requesting an own Token as an anonymous user
+	Given the user is unauthenticated
+	And an own Token t
+	When a GET request is sent to the Tokens/{id} endpoint with t.Id
+	Then the response status code is 200 (OK)
+	And the response contains a Token
+
+Scenario: Requesting a peer Token as an authenticated user
+	Given the user is authenticated
+	And a peer Token p
+	When a GET request is sent to the Tokens/{id} endpoint with p.Id
+	Then the response status code is 200 (OK)
+	And the response contains a Token
+
+Scenario: Requesting a list of own Tokens
+	Given the user is authenticated
+	And the user created multiple Tokens
+	When a GET request is sent to the Tokens endpoint with the Tokens list
+	Then the response status code is 200 (OK)
+	And the response does not contain any peer Tokens
+
```

Comment:
There should be a test where we have a Given that makes sure there is 1 own Token and 1 peer Token. Then we pass the id of both. As a result, we expect to only receive the own Token.